src/frontend-helper.ts

```diff
@@ -112,3 +129,51 @@ const setImageStatus = (images: Image[], status: IMAGE_STATUS) => {
     );
   });
 };
+
+class policyStatusByAdGroup {
```

Comment:
nit: classes should be in UpperCamelCase
```suggestion
class PolicyStatusByAdGroup {
```